app/controllers/ansible_repository_controller.rb

```diff
@@ -48,10 +48,20 @@ def button
         page.replace("gtl_div", :partial => "layouts/gtl")
       end
     when "ansible_repository_reload" # repository reload
-      show
-      render :update do |page|
-        page << javascript_prologue
-        page.replace("main_div", :template => "ansible_repository/show")
+      if @display == "output"
+        show
+        show_output
+        @display = "output" # reset @display back to "output" after show changes it to "main"
```

Comment:
@GilbertCherrie , do we need this line? `@display `is already `output` here.

Reply:
Yeah we do, since the show function changes it back to "main"